Assets/Packages/com.studio23.ss2.upmassistant/Editor/Data/License.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Studio23.SS2.UPMAssistant.Editor.Data
+{
+    [System.Serializable]
+    public class License
+    {
+        public string key;
```

Comment:
Naming convention is not correct. For public variable it should be 'public string Key'